packages/openneuro-app/src/sass/crn_app/_dashboard.datasets.scss

```diff
@@ -187,6 +188,26 @@
     }
 }
 
+@media only screen and (max-width: 765px) {
```

Comment:
We use 765px as the breakpoint in many places in the code. I think it would be valuable to pull this out into a variable with a descriptive name like like $tablet-width or something.